### PR TITLE
fix: change initial state banner and button texts

### LINKED ISSUE #4905: [bugs] verified users
As an admin I want to see what the state of a user is, so I can help them on support requests.

**Acceptance Criteria**
- [ ] If user is initial state, I see a banner "User is not active yet. Button: Resend Initial Mail"

**Additional Context**

If I create a user with verified email I get a warning that the users email is not verified, but It is verified, the user is just initial.

## Detail view:
image

## List view
image

### PR DESCRIPTION
**Pull request by doncicuto**
Messages for banner and buttom have been changed as @adlerhurst has suggested.

Screenshot of new texts:

initial_user_state

Acceptance Criteria

- [X] If user is initial state, I see a banner "User is not active yet. Button: Resend Initial Mail"

Closes #4905 

```[tasklist]
### Definition of Ready
- [X] I am happy with the code
- [X] Short description of the feature/issue is added in the pr description
- [X] PR is linked to the corresponding user story
- [X] Acceptance criteria are met
- [ ] All open todos and follow ups are defined in a new ticket and justified
- [ ] Deviations from the acceptance criteria and design are agreed with the PO and documented.
- [ ] No debug or dead code
- [ ] Critical parts are tested automatically
- [ ] Where possible E2E tests are implemented
- [ ] Documentation/examples are up-to-date
- [ ] All non-functional requirements are met
- [X] Functionality of the acceptance criteria is checked manually on the dev system.
```

### CODE REVIEW

File: console/src/assets/i18n/it.json [hunk 1]
```diff
@@ -612,7 +612,7 @@
     "SIGNEDOUT_BTN": "Accedi",
     "EDITACCOUNT": "Modifica account",
     "ADDACCOUNT": "Accedi con un altro account",
-    "RESENDINITIALEMAIL": "Invia una nuova mail di inizializzazione",
+    "RESENDINITIALEMAIL": "Invia nuovamente la posta iniziale",
```
Comment (peintnermax) [inline console/src/assets/i18n/it.json:615]:
```suggestion
    "RESENDINITIALEMAIL": "Inviare nuovamente l'e-mail di avvio",
```

File: console/src/assets/i18n/de.json [hunk 2]
```diff
@@ -612,7 +612,7 @@
     "SIGNEDOUT_BTN": "Anmelden",
     "EDITACCOUNT": "Konto bearbeiten",
     "ADDACCOUNT": "Konto hinzufügen",
-    "RESENDINITIALEMAIL": "Neue Initialisierungsmail senden",
+    "RESENDINITIALEMAIL": "Ursprüngliche E-Mail erneut senden",
```
Comment (peintnermax) [inline console/src/assets/i18n/de.json:615]:
```suggestion
    "RESENDINITIALEMAIL": "Init-Mail erneut senden",
```

**Review by peintnermax — COMMENTED** [review]
@adlerhurst @eliobischof other opinions?

**Comment by eliobischof** [comment]
Let's try to stick to a consistent terminology?
Eighter like
- "User is not initialized, yet" and "Resend initialization mail"
or
- "User is not active, yet" and "Resend activation mail"

**Comment by peintnermax** [comment]
> Let's try to stick to a consistent terminology? Eighter like
> 
> * "User is not initialized, yet" and "Resend initialization mail"
>   or
> * "User is not active, yet" and "Resend activation mail"

like the proposal.. prefer the second one actually :)

**Comment by doncicuto** [comment]
Hi @peintnermax if you want to me to do any change let me know

**Comment by peintnermax** [comment]
I've updated the commit suggestions above. Can you set them for the other languages too? 
I'd stick with @eliobischof proposal. Thx for your work 👍 
en: `Resend activation mail`

**Comment by doncicuto** [comment]
Absolutely @peintnermax, I'll prepare the new PR according to @eliobischof's proposal

**Review by peintnermax — APPROVED** [review]

**Pull request merged**